docker-config/dev.yml

```diff
@@ -1,147 +0,0 @@
-version: '2'
```

Comment:
As per the original comment, we're no longer building the images here - but we still need to be able to run a dev environment for openspending.

Deleting everything is throwing the baby out with the bathwater - we need to adjust the docker-compose.yml with the latest updates that might have been added in production so that it runs using the prebuilt images.